src/index.ts

```diff
@@ -105,11 +105,15 @@ const backportPullRequest = async ({
       debug("commits cherry-picked", headSha);
     } catch (error) {
       debug("commits could not be cherry-picked", error);
-      throw new Error(
+      const newError = new Error(
         `Commits ${JSON.stringify(
           commits,
         )} could not be cherry-picked on top of ${base}`,
       );
+      if(error.name === "HttpError") {
+        newError.httpError = error
+      }
+      throw newError;
```

Comment:
Or perhaps simply re-throw the original error:
```suggestion
      throw error;
```